reflex/components/radix/primitives/drawer.py

```diff
@@ -142,6 +143,25 @@ def get_event_triggers(self) -> Dict[str, Any]:
             EventTriggers.ON_INTERACT_OUTSIDE: lambda e0: [e0.target.value],
         }
 
+    @classmethod
+    def create(cls, *children, **props):
+        """Create a Drawer Content.
+         We wrap the Drawer content in an `rx.theme` to make radix themes definitions available to
+         rendered div in the DOM. This is because Vaul Drawer injects the Drawer overlay content in a sibling
+         div to the root div rendered by radix which contains styling definitions. Wrapping in `rx.theme`
+         makes the styling available to the overlay.
+
+        Args:
+            *children: The list of children to use if header and content are not provided.
+            **props: Additional properties to apply to the accordion item.
+
+        Returns:
+                 The drawer content.
+        """
+        comp = super().create(*children, **props)
+
+        return Theme.create(comp)
```

Comment:
does this actually work though? if the user customizes the values in their app theme, doesn't seem like those would apply to the drawer content

Reply:
Yeah I think it does work. The rendered code looks like below. The global themes are hence inherited, unless theres something else Im missing.

```javascript
...
export default function Component() {

  return (
    <Fragment>
      <Fragment_1762bb90abdb81b879b2a22edbbe01a1 />
      <Fragment>
        <VaulDrawer.Root direction={`left`}>
          <VaulDrawer.Trigger asChild={true}>
            <RadixThemesButton>
              {`Open Drawer`}
            </RadixThemesButton>
          </VaulDrawer.Trigger>
          <VaulDrawer.Overlay css={{ "position": "fixed", "left": "0", "right": "0", "bottom": "0", "top": "0", "zIndex": 50, "background": "rgba(0, 0, 0, 0.5)" }} />
          <VaulDrawer.Portal>
            <RadixThemesTheme css={{ ...theme.styles.global[':root'], ...theme.styles.global.body }}>
              <VaulDrawer.Content css={{ "left": "0", "right": "auto", "bottom": "0", "top": "auto", "position": "fixed", "zIndex": 50, "display": "flex", "height": "100%", "width": "20em", "padding": "2em" }} height={`100%`} padding={`2em`} right={`auto`} top={`auto`} width={`20em`}>
                <RadixThemesFlex css={{ "alignItems": "start" }} direction={`column`}>
                  <RadixThemesButton>
                    {`another button`}
                  </RadixThemesButton>
                  <VaulDrawer.Close>
                    {`Close`}
                  </VaulDrawer.Close>
                </RadixThemesFlex>
              </VaulDrawer.Content>
            </RadixThemesTheme>
          </VaulDrawer.Portal>
        </VaulDrawer.Root>
      </Fragment>
      <NextHead>
        <title>
          {`Reflex App`}
        </title>
        <meta content={`A Reflex app.`} name={`description`} />
        <meta content={`favicon.ico`} property={`og:image`} />
      </NextHead>
    </Fragment>
  )
}

```